File: CustomComponents/app/src/main/java/com/vibeosys/customcomponents/MainActivity.java
package com.vibeosys.customcomponents;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.vibeosys.shuttercomponent.ShutterComponent;
import com.vibeosys.shuttercomponent.ShutterHeader;

public class MainActivity extends AppCompatActivity {

    ShutterHeader component;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        component = (ShutterHeader) findViewById(R.id.txtHeader);
        component.setTxtType("INR TO USD");
        component.setTxtPrice("0.84563");
        component.setTxtBuyOrSellColor(getResources().getColor(R.color.accentText));
    }
}

File: app/src/main/java/com/vibeosys/customutils/MainActivity.kt
package com.vibeosys.customutils

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.customutils.*
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        if(!NetworkUtils.isNetworkAvailable(this))
        {
            CustomToast(this).showNetworkUnAvailable()
        }
        loginBtn.setOnClickListener {
            if(emailEt.isValid() && nameEt.isValid())
            {

            }

        }

    }
}
